extensions/roc-package-web-app-react/src/app/server/reactRenderer.js

```diff
@@ -30,18 +31,42 @@ const whiteListed = () => (
 
 const appConfig = whiteListed();
 
-export function initRenderPage({ script, css }, distMode, devMode, Header) {
+function setupTemplate(devMode) {
     const templatePaths = [].concat(
         // Combine paths from highest priority to lowest
         rocConfig.runtime.template.path || [],
         invokeHook('get-template-paths'),
```

Comment:
I know that this was not done in this PR but does it make sense that the action is responsible for taking the previous value into consideration? The scenario that I see it that if an extension want to remove a added path but that scenario seems unlikely, should we move the concat functionality to a callback instead?